### PR TITLE
feat: added in bindings to updated jx-health module that includes tf drift check and so requires git bot creds

### PR DESCRIPTION
**Pull request by chrismellard**

### CODE REVIEW

File: modules/jx/charts.tf [hunk 1]
```diff
@@ -35,9 +35,23 @@ resource "helm_release" "jx-git-operator" {
...
+    "ARM_CLIENT_ID" : var.jx_health_client_id,
+    "ARM_CLIENT_SECRET" : var.jx_health_client_secret,
+  }
+}
+
 module "jx-health" {
-  count  = var.enabled ? 1 : 0
-  source = "github.com/jenkins-x/terraform-jx-health?ref=main"
+  count               = var.enabled ? 1 : 0
+  source              = "/Users/chrismellard/go/src/github.com/jenkins-x/terraform-jx-health"
```
Comment (hervelemeur) [inline modules/jx/charts.tf]: Should it be a public url instead of a local file path?

Comment (chrismellard) [reply modules/jx/charts.tf]: thanks @hervelemeur - tearing my hair out wondering why the local tests worked but didn't remotely ;)

**Pull request closed without merging**